api/turing/cluster/knative_service_test.go

```diff
@@ -135,7 +135,7 @@ func TestBuildKnativeServiceConfig(t *testing.T) {
 				},
 				Resources: resources,
 				LivenessProbe: &corev1.Probe{
-					Handler: corev1.Handler{
+					ProbeHandler: corev1.ProbeHandler{
```

Comment:
Similar to https://github.com/caraml-dev/merlin/pull/551/files#r1521016971, this field has been renamed.